src/redux/actions/internalActions.js

```diff
@@ -242,6 +242,10 @@ export const setMouseWheelZoom = (enableMouseWheelZoom = true) => ({
   type: 'SET_MOUSE_WHEEL_ZOOM',
   payload: { enableMouseWheelZoom },
 });
+export const setMathSymbols = (enableMathSymbols = true) => ({
```

Comment:
As discussed in the screenshare before we actually don't need to add an action, reducer, and a selector for toggling the math symbols button. Since the code isn't necessary I think we should remove them.

Reply:
I removed the changes from action and reducer, but I guess we still need the update inside `src/redux/selectors/exposedSelectors.js`, can you please take another look? @ZhijieZhang